spotbugs/etc/findbugs.xml

```diff
@@ -1320,4 +1324,5 @@
           <BugPattern abbrev="PI" type="PI_DO_NOT_REUSE_PUBLIC_IDENTIFIERS_METHOD_NAMES" category="BAD_PRACTICE"/>
           <BugPattern abbrev="PI" type="PI_DO_NOT_REUSE_PUBLIC_IDENTIFIERS_LOCAL_VARIABLE_NAMES" category="BAD_PRACTICE"/>
 
+          <BugPattern abbrev="HSBC" type="HSBC_HIDING_SUB_CLASS" category="CORRECTNESS"/>
```

Comment:
I think it will be more descriptive if you change the type to `HSM_HIDING_METHOD`, and the abbrev will be `HSM` in this case.